.github/workflows/publish.yml

```diff
@@ -72,8 +72,12 @@ jobs:
           npm install pnpm --global
           pnpm install
           pnpm build
-      - name: Tests
-        run: ./gradlew server:test
+      - name: Build & Test Server
```

Comment:
Building the jar as part of the pipeline drastically reduces the amount of time the `docker build` for the `linux/arm64` platform. Also, this same jar could be leverage for the `lh-sever` image instead of building it there too.

Reply:
This seems like a good idea, I like it.